Installation/cmake/modules/CGAL_SetupCGALDependencies.cmake

```diff
@@ -144,6 +144,12 @@ function(CGAL_setup_CGAL_dependencies target)
     if ( RUNNING_CGAL_AUTO_TEST )
       target_compile_options(${target} INTERFACE "-Wall")
     endif()
+    if(NOT CGAL_HAS_NO_THREADS)
```

Comment:
I think using CGAL_HAS_NO_THREADS as a CMake thing is new. Should there be an `else` that defines CGAL_HAS_NO_THREADS as a macro?
We are going to need some doc...

Reply:
I agree with you:
  - `CGAL_HAS_NO_THREADS` as a preprocessor macro is currently documented in the [section "Thread Safety"](https://doc.cgal.org/5.2.2/Manual/preliminaries.html#title1) in the manual. The eponymous CMake option should be documented.
  - I agree the CMake option `CGAL_HAS_NO_THREADS`, if true, should trigger a `-DCGAL_HAS_NO_THREADS` in compiler options.

I will fix that...

Reply:
I have tried to fix that with commit https://github.com/CGAL/cgal/pull/5402/commits/530616534ad9c68954067b4722d65d6fba217ec2.